adminutils/adminutils.py

```diff
@@ -267,6 +295,8 @@ async def emoji_rename(self, ctx, emoji: discord.Emoji, name: str, *roles: disco
             )
         except discord.Forbidden:
             await ctx.send(chat.error(_("I can't edit this emoji")))
+        else:
+            await ctx.send(_(f"{em} edited to `{em.name}`."))
```

Comment:
f-strings are not gonna work with i18n, please use `_("{} string {}).format(var, var)`